src/features/item-details/ItemProperties.js

```diff
@@ -116,6 +126,8 @@ const ItemProperties = ({
 };
 
 ItemProperties.propTypes = {
+    /** the datetime this item was archived, accepts any value that can be passed to the Date() constructor */
+    archivedAt: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
can it actually be a number?

Reply:
Changed to string

Reply:
Changed again since we're probably are going to be using timestamps